microsite/data/plugins/kubelog.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Kubelog
+author: Julio Fernandez
+authorUrl: https://github.com/jfvilasPersonal
+category: Observability
+description: Kubelog provides your BackStage users the ability to view logs of your Kubernetes objects linked to BackStage entities. Security controls can be applied.
+documentation: https://github.com/jfvilasPersonal/kubelog
+iconUrl: https://raw.githubusercontent.com/jfvilasPersonal/kubelog/main/src/assets/kubelog-logo.png
+npmPackageName: '@jfvilas/plugin-kubelog'
```

Comment:
Totally optional but it would be nice to have your NPM scope match your GitHub Org name, just makes people less likely to 2nd guess the sources when looking at the package. You do have a link tot he repo so that should help with that.

Reply:
I agree. I re-styled Backstage name in all components and move all the code to a github repo that matches the npm scope. I'll close this PR and create a new one with my old-new username (jfvilas) and the repos moved to this space. Thanks!!